pubspec.lock

```diff
@@ -155,6 +155,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "0.9.4"
+  firebase_storage:
+    dependency: "direct main"
+    description:
+      name: firebase_storage
+      url: "https://pub.dartlang.org"
+    source: hosted
+    version: "3.1.6"
```

Comment:
Thanks for thinking about using Firebase Storage to store the cover images. This is brilliant.

Reply:
Thanks